templates/c2s-lb/config.yml

```diff
@@ -1,6 +1,6 @@
 name: C2S - Load Balancer
 description: |
   Connecting to Services - Load Balancer config for staging/production
-version: v1
-category: nhsuk beta
+version: 1.0.0
+category: nhsuk beta load-balancer
 projectURL: https://github.com/nhsuk/nhsuk-rancher-templates
```

Comment:
That version relates to the one in `rancher-compose` e.g. https://github.com/nhsuk/nhsuk-rancher-templates/blob/master/templates/c2s-lb/0/rancher-compose.yml#L5
The `v1` didn't relate to anything